src/torus.ts

```diff
@@ -171,10 +172,10 @@ class Torus {
   async getPublicAddress(
     endpoints: string[],
     torusNodePubs: INodePub[],
-    { verifier, verifierId, extendedVerifierId }: { verifier: string; verifierId: string; extendedVerifierId?: string }
+    { verifier, verifierId, extendedVerifierId, keyType }: { verifier: string; verifierId: string; extendedVerifierId?: string; keyType?: KeyType }
   ): Promise<TorusPublicKey> {
     log.info(torusNodePubs, { verifier, verifierId, extendedVerifierId });
-    return this.getNewPublicAddress(endpoints, { verifier, verifierId, extendedVerifierId }, this.enableOneKey);
+    return this.getNewPublicAddress(endpoints, { verifier, verifierId, extendedVerifierId, keyType }, this.enableOneKey);
```

Comment:
can we pass keyType here as `const localKeyType = keyType ?? this.keyType` for the sake of consistency with other public functions